src/sync-client.js

```diff
@@ -920,6 +925,8 @@ var self = {
     });
   },
 
+  getHashMethod: CryptoJS.SHA1,
```

Comment:
👎  We exposing CryptoJS interface without telling people what it is. Antipattern